i18n/en.json

```
@@ -268,7 +268,7 @@
   "admin.billing.subscription.cancelSubscriptionSection.description": "At this time, deleting a workspace can only be done with the help of a customer support representative.",
   "admin.billing.subscription.cancelSubscriptionSection.title": "Cancel your subscription",
   "admin.billing.subscription.cloudTrial.daysLeftOnTrial": "There are {daysLeftOnTrial} days left on your free trial",
-  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire today at 11:59pm PT.",
+  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire at {userEndTrialHour} PT.",
```

We should add the users timezone in the time/date formatting and remove this hardcoded `Pacific Time` reference
```suggestion
  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire at {userEndTrialHour}.",
```

---

ok!